javaparser-core/src/main/java/com/github/javaparser/ast/observer/ObservableProperty.java

```diff
@@ -35,7 +35,7 @@
IPLE_REFERENCE), CHECK(Type.SINGLE_REFERENCE), CLASS_BODY(Type.MULTIPLE_REFERENCE), CLASS_DECLARATION(Type.SINGLE_REFERENCE), CLASS_EXPR(Type.SINGLE_REFERENCE), COMMENT(Type.SINGLE_REFERENCE), COMPARE(Type.SINGLE_REFERENCE), COMPONENT_TYPE(Type.SINGLE_REFERENCE), CONDITION(Type.SINGLE_REFERENCE), CONTENT(Type.SINGLE_ATTRIBUTE), DEFAULT_VALUE(Type.SINGLE_REFERENCE), DIMENSION(Type.SINGLE_REFERENCE), ELEMENTS(Type.MULTIPLE_REFERENCE), ELEMENT_TYPE(Type.SINGLE_REFERENCE), ELSE_EXPR(Type.SINGLE_REFERENCE), ELSE_STMT(Type.SINGLE_REFERENCE), ENCLOSING_PARAMETERS(Type.SINGLE_ATTRIBUTE), ENTRIES(Type.MULTIPLE_REFERENCE), EXPRESSION(Type.SINGLE_REFERENCE), EXTENDED_TYPE(Type.SINGLE_REFERENCE), EXTENDED_TYPES(Type.MULTIPLE_REFERENCE), FINALLY_BLOCK(Type.SINGLE_REFERENCE), IDENTIFIER(Type.SINGLE_ATTRIBUTE), IMPLEMENTED_TYPES(Type.MULTIPLE_REFERENCE), IMPORTS(Type.MULTIPLE_REFERENCE), INDEX(Type.SINGLE_REFERENCE), INITIALIZATION(Type.MULTIPLE_REFERENCE), INITIALIZER(Type.SINGLE_REFERENCE), INNER(Type.SINGLE_REFERENCE), INTERFACE(Type.SINGLE_ATTRIBUTE), ITERABLE(Type.SINGLE_REFERENCE), LABEL(Type.SINGLE_REFERENCE), LEFT(Type.SINGLE_REFERENCE), LEVELS(Type.MULTIPLE_REFERENCE), MEMBERS(Type.MULTIPLE_REFERENCE), MEMBER_VALUE(Type.SINGLE_REFERENCE), MESSAGE(Type.SINGLE_REFERENCE), MODIFIERS(Type.MULTIPLE_ATTRIBUTE), MODULE(Type.SINGLE_REFERENCE), MODULE_NAMES(Type.MULTIPLE_REFERENCE), MODULE_STMTS(Type.MULTIPLE_REFERENCE), NAME(Type.SINGLE_REFERENCE), OPEN(Type.SINGLE_ATTRIBUTE), OPERATOR(Type.SINGLE_ATTRIBUTE), PACKAGE_DECLARATION(Type.SINGLE_REFERENCE), PAIRS(Type.MULTIPLE_REFERENCE), PARAMETER(Type.SINGLE_REFERENCE), PARAMETERS(Type.MULTIPLE_REFERENCE), QUALIFIER(Type.SINGLE_REFERENCE), RESOURCES(Type.MULTIPLE_REFERENCE), RIGHT(Type.SINGLE_REFERENCE), SCOPE(Type.SINGLE_REFERENCE), SELECTOR(Type.SINGLE_REFERENCE), STATEMENT(Type.SINGLE_REFERENCE), STATEMENTS(Type.MULTIPLE_REFERENCE), STATIC(Type.SINGLE_ATTRIBUTE), STATICNESS(Type.SINGLE_ATTRIBUTE), SUPER_TYPE(Type.SINGLE_REFERENCE), TARGET(Type.SINGLE_REFERENCE), THEN_EXPR(Type.SINGLE_REFERENCE), THEN_STMT(Type.SINGLE_REFERENCE), THIS(Type.SINGLE_ATTRIBUTE), THROWN_EXCEPTIONS(Type.MULTIPLE_REFERENCE), TRY_BLOCK(Type.SINGLE_REFERENCE), TYPE(Type.SINGLE_REFERENCE), TYPES(Type.MULTIPLE_REFERENCE), TYPE_ARGUMENTS(Type.MULTIPLE_REFERENCE), TYPE_BOUND(Type.MULTIPLE_REFERENCE), TYPE_PARAMETERS(Type.MULTIPLE_REFERENCE), UPDATE(Type.MULTIPLE_REFERENCE), VALUE(Type.SINGLE_REFERENCE), VALUES(Type.MULTIPLE_REFERENCE), VARIABLE(Type.SINGLE_REFERENCE), VARIABLES(Type.MULTIPLE_REFERENCE), VAR_ARGS(Type.SINGLE_ATTRIBUTE), VAR_ARGS_ANNOTATIONS(Type.MULTIPLE_REFERENCE), WITH_TYPES(Type.MULTIPLE_REFERENCE), ELSE_BLOCK(Type.SINGLE_ATTRIBUTE, true), EXPRESSION_BODY(Type.SINGLE_REFERENCE, true), MAXIMUM_COMMON_TYPE(Type.SINGLE_REFERENCE, true), POSTFIX(Type.SINGLE_ATTRIBUTE, true), PREFIX(Type.SINGLE_ATTRIBUTE, true), THEN_BLOCK(Type.SINGLE_ATTRIBUTE, true), USING_DIAMOND_OPERATOR(Type.SINGLE_ATTRIBUTE, true), RANGE, COMMENTED_NODE;
```

Comment:
Maybe we should add newlines to make easier spotting differences

Reply:
This is what our pretty printer does :-/